.gitignore

```diff
@@ -0,0 +1 @@
+/secret/
```

Comment:
I use it to keep state/var-files in for my many test stacks so I don't have to repeatedly stash them.